File: browsergameMapEditor/models/MapTile.py
from models.Color import Color
from models.JSONAble import JSONAble


class MapTile(JSONAble):
    def __init__(self, x: int, y: int, region: int, color: Color):
        super().__init__()
        self.x = x
        self.y = y
        self.region = region
        self.color = color

File: BrowserGameServer/BrowserGameServer/GameServer/Map/MapTile.cs
using System.Drawing;
using System.Numerics;

namespace BrowserGameServer.GameServer.Map;

public struct MapTile
{
    public int x { get; set; }
    public int y { get; set; }
    public int region { get; set; }
    public Color color { get; set; }

    public MapTile(int x, int y, int region, Color color)
    {
        this.x = x;
        this.y = y;
        this.region = region;
        this.color = color;
    }
}